README.md

```diff
@@ -40,13 +40,15 @@ Usage: atomizer [options] [path]
     -H, --helpersNamespace [namespace]  adds the given namespace to all helper selectors.
     --rtl                               swaps `start` and `end` keyword replacements with `right` and `left`.
     --ie                                adds old IE hacks to the output.
+    --exclude [pattern]                 pattern to exclude file to be scanned
```

Comment:
Probably should add the `=` here too

Reply:
Actually all of the README examples need it, I'll do this seperately

Reply:
@src-code it seems like adding `=` then it doesn't work anymore.

program.exclude is `undefined`

https://www.npmjs.com/package/commander

if I remove equal then it works
`.option('--exclude [pattern]', 'excluded file pattern', collect, [])`

Reply:
Ah, interesting.  Ok, will merge your other PR